app/src/main/java/com/example/remindmelater/MainActivity.kt

```diff
@@ -132,14 +135,14 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
                         .padding(4.dp)
                         .height(35.dp)
                         .width(190.dp),
-                    colors = ButtonDefaults.buttonColors(backgroundColor = Color(12, 121, 230))
+                    colors = ButtonDefaults.buttonColors(backgroundColor = buttonColor)
```

Comment:
DRY principle 

Added a hex code for same color in color resource, pulled that hex color in local variable and called it here